lib/msal-browser/src/config/Configuration.ts

```diff
@@ -14,6 +14,7 @@ export type BrowserAuthOptions = {
     clientId: string;
     authority?: string;
     knownAuthorities?: Array<string>;
+    instanceMetadata?: Array<IInstanceDiscoveryMetadata>;
```

Comment:
Naming here might get confusing when we implement the other perf changes which introduced `authorityMetadata` in msal-core. Open to suggestions for a better name here.